EasyLearn/app/assets/stylesheets/styles.css

```diff
@@ -0,0 +1,145 @@
+.contenido{
+  background-color: #282828;
+  font-family: 'Montserrat', sans-serif;
+}
+
+.nav{
+  align-items: center;
+  color: white;
+  display: flex;
+  justify-content: space-between;
+  margin: 0 auto;
+  width: 80%;
+}
+ 
+.nav__items{
```

Comment:
If they want use this sintax it is specific to use on bem nomeclature with sass, this clas name can be better how `.nav-items` in css